usr/share/rear/output/PXE/default/801_copy_to_http.sh

```diff
@@ -0,0 +1,55 @@
+# 801_copy_to_http.sh
+#
+# copy kernel and initrd to HTTP server for Relax-and-Recover
+#
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+# Nothing to do here when PXE_HTTP_UPLOAD_URL is not set
+if [[ -z "$PXE_HTTP_UPLOAD_URL" ]] ; then
+    LogPrint "PXE_HTTP_UPLOAD_URL not set, skipping copy"
```

Comment:
I think this should be only a DebugPrint message
because PXE_HTTP_UPLOAD_URL is not set by default
so the user would get this message by default (in verbose mode)
but I think it is not needed to tell the user about things
that are disabled by default.

Reply:
I agree after considering expectation management: there might be a situation where PXE_HTTP_UPLOAD_URL is NOT set but PXE_HTTP_DOWNLOAD_URL is set. This situation means that the user configured (and expects) an http download, but did not configure an upload. But this is handled already in `./usr/share/rear/init/default/002_pxe_sanity_checks.sh`.

Fixed with 622c548679e045cf00c5aacd3ee51b76a614a77d.